windowsforms/FlexGridShowcaseDemo/cs/README.md

```diff
@@ -0,0 +1,50 @@
+---
+languages:
+- csharp
+products:
+- dotnet
+- dotnet-core
+- windows-forms
+page_type: sample
+name: "Windows Forms Sample: FlexGrid Showcase (C#)"
+urlFragment: "winforms-flexgridshowcase-cs"
+description: ".NET 5.0 Windows Forms application that demonstrates how to use GrapeCity FlexGrid"
+---
+
+# FlexGrid Showcase Demo
+
+This sample is a .NET 5.0 Windows Forms application that demonstrates how to use GrapeCity [`FlexGrid`](https://www.grapecity.com/componentone/docs/win/online-flexgrid/overview.html).
+
+The sample shows different operations with mentioned controls:
+
+* Search
+* Formatting
+* Data grouping
+
+![Screenshot with applied conditional formatting](../images/screenshot1.png)
+
+![Screenshot with grouped data](../images/screenshot2.png)
+
+## Sample prerequisites
+
+This sample is written in C# and targets .NET 5.0 running on Windows. It requires the [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0).
+
+## Building the sample
+
+To download and run the sample, follow these steps:
```

Comment:
Since this depends on a GrapeCity FlexGrid, there has to be step explaining where to get it and how to install.

Reply:
Actually the sample depends on several our packages, all of these are available on nuget.org - https://www.nuget.org/packages/C1.Win.FlexGrid/5.0.20202.439-beta, https://www.nuget.org/packages/C1.Win.Ribbon/5.0.20202.439-beta. etc.
They referenced as package references in the project.
What do you think - should I explain how to install it from nuget.org?
I think the installation from nuget might be not just one simple step and I'm afraid to confuse users.
Could you suggest me how to explain it simply?
Thanks.

Reply:
> What do you think - should I explain how to install it from nuget.org?
> I think the installation from nuget might be not just one simple step and I'm afraid to confuse users.

I don't think it is necessary to explain how to install from NuGet, but I feel that it should be explicitly called out that the sample depends on GrapeCity NuGet packages (provide a list). 
I also think it is important to be explicit whether the control is paid, free, or trial-based.

Reply:
I made changes in README as you suggested.
Please, take a look - 6bfd5764fd3cf86e613b4f56a3c4eec9bcd18bda.